Lesson_07/l04_h01_integer_to_english_words.py

```diff
@@ -0,0 +1,31 @@
+"""
+HW:7[Volkov Anton].
+
+Convert a non-negative integer num to its English words representation.
+Example 1:
+Input: num = 123
+Output: 'One Hundred Twenty Three'
+let's take that number always > 100 and only three digits
+100 <= num <= 999
+"""
+num = 123
+number = ['', 'One', 'Two', 'Three', 'Four', 'Five',
```

Comment:
Better organize such structures as dictionaries and then in code use dict.get(digit_00)
Dictionaries much easier to read than find certain element by index in list manually )